flink-runtime-web/web-dashboard/app/styles/index.styl

```diff
@@ -405,3 +405,7 @@ pre.exception
   .table
     td.td-long
       width: 30%
+
+a
```

Comment:
This is kind of required to make the mouse change to pointer on hovering over the refresh button. But yes, will make it a little better.